projects/netgrif-components-core/src/lib/view/tree-case-view/tree-component/model/tree-case-view-configuration.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Contains configuration of the tree case view service
+ */
+export interface TreeCaseViewConfiguration {
```

Comment:
It would be better to have a common paging interface with page, size, sort params.

Reply:
this is currently the only interface used for configuration objects that handles pagination configuration by developers.